.github/workflows/test.yml

```diff
@@ -217,9 +450,186 @@ jobs:
             !/home/runner/pytest_workflow_*/*/work/singularity
             !${{ github.workspace }}/.singularity
 
+  nf-test:
+    runs-on: ubuntu-latest
+    name: nf-test
+    needs: [nf-test-changes]
+    if: needs.nf-test-changes.outputs.modules != '[]'
+    strategy:
+      fail-fast: false
+      matrix:
+        tags: ["${{ fromJson(needs.nf-test-changes.outputs.modules) }}"]
+        profile: ["docker", "singularity", "conda"]
+        exclude:
+          - tags: "nf-test"
+          - tags: "nf-test"
+          - profile: "conda"
+            tags: annotsv
+          - profile: "conda"
+            tags: happy/sompy
+          - profile: "conda"
+            tags: backsub
+          - profile: "conda"
+            tags: bases2fastq
+          - profile: "conda"
+            tags: bcl2fastq
+          - profile: "conda"
+            tags: bclconvert
+          - profile: "conda"
+            tags: cellranger/count
+          - profile: "conda"
+            tags: cellranger/mkfastq
+          - profile: "conda"
+            tags: cellranger/mkgtf
+          - profile: "conda"
+            tags: cellranger/mkref
+          - profile: "conda"
+            tags: custom/dumpsoftwareversions
+          - profile: "conda"
+            tags: deepcell/mesmer
+          - profile: "conda"
+            tags: deepvariant
+          - profile: "conda"
+            tags: ensemblvep/vep
+          - profile: "conda"
+            tags: fastk/fastk
+          - profile: "conda"
+            tags: fastk/histex
+          - profile: "conda"
+            tags: fastk/merge
+          - profile: "conda"
+            tags: fcs/fcsadaptor
+          - profile: "conda"
+            tags: fcs/fcsgx
+          - profile: "conda"
+            tags: gatk4/cnnscorevariants
+          - profile: "conda"
+            tags: gatk4/determinegermlinecontigploidy
+          - profile: "conda"
+            tags: genescopefk
+          - profile: "conda"
+            tags: ilastik/multicut
+          - profile: "conda"
+            tags: ilastik/pixelclassification
+          - profile: "conda"
+            tags: imputeme/vcftoprs
+          - profile: "conda"
+            tags: merquryfk/katcomp
+          - profile: "conda"
+            tags: merquryfk/katgc
+          - profile: "conda"
+            tags: merquryfk/merquryfk
+          - profile: "conda"
+            tags: merquryfk/ploidyplot
+          - profile: "conda"
+            tags: sentieon/bwaindex
+          - profile: "conda"
+            tags: sentieon/bwamem
+          - profile: "conda"
+            tags: svanalyzer/svbenchmark
+          - profile: "conda"
+            tags: universc
+          - profile: "singularity"
+            tags: universc
+          - profile: "conda"
+            tags: vt/decompose
+          - profile: "conda"
+            tags: subworkflows/vcf_annotate_ensemblvep
```

Comment:
do you plan to update the conda test? if not we should we should probably exclude the CI test by adding it here. i think we can just add something like 
```
- profile: "conda"
  tags: gbcms
```

Reply:
No plans as of right now. I added conda gbcms to the exclude block. 

I also removed the nf-core modules from the exclude block.